pom.xml

```diff
@@ -13,10 +13,10 @@
     </parent>
 
     <groupId>org.jboss.windup</groupId>
-    <artifactId>windup-distribution</artifactId>
+    <artifactId>rhamt-distribution</artifactId>
```

Comment:
well, if this will be kept, then the assembled zip doesn't make sense to me as I will overwrite artifact id which is normally in assembly archive.